challenge.c

```diff
@@ -2,28 +2,31 @@
 #include <stdbool.h>
 #include <stdlib.h>
 #include <stdio.h>
+#include <inttypes.h>
```

Comment:
Is this header used somewhere in this file?

Reply:
No, it's not. I think I wanted to use a function from there at some point, but I ended up not requiring it and forgot to remove it , sorry.